### DISCUSSION TITLE
Option to add last changed xy minutes ago into Title

### DISCUSSION THREAD

**Discussion by juliuspollak** [dylandoamaral/uptime-card#155] [General]
Hi,

would it possible to add last updated information from a sensor into the Title? 

.. If i click on sensor's history there is a timestamp and xy minutes ago information which I would like to have in the title if possible.

Thank you

![gate](https://github.com/dylandoamaral/uptime-card/assets/99386240/bfcea63a-cdb0-4047-88aa-9c921a6d3960)
![gate_2](https://github.com/dylandoamaral/uptime-card/assets/99386240/cacc4671-6778-4f71-bfb3-b499e269316a)

**Comment by dylandoamaral** [comment]
Yes you should look at https://github.com/dylandoamaral/uptime-card#templating for more information.
